spec/venue_spec.rb

```diff
@@ -43,13 +38,56 @@
   end
 
   describe '#yell_at_patrons' do
-    it 'returns a list of uppercased names' do
-      skip
+    it 'returns a list of uppercased patron names' do
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+
+  # Iteration 3
+
+  describe '#over_capacity?' do
+    it 'returns false because venue is not over capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      expect(venue.over_capacity?).to eq false
+    end
+  end
+
+  describe '#over_capacity?' do
+    it 'returns true because venue is over capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      expect(venue.over_capacity?).to eq true
+    end
+  end
+
+
+
+  # Iteration 4
+
+  describe '#kick_out' do
+    it 'kicks out patrons until less than or equal to capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      venue.kick_out
+      expect(venue.patrons.count).to be <= venue.capacity
```

Comment:
Make sure to follow the interaction pattern. It expected this:
```
pry(main)> venue.over_capacity?
=> false
```
So that would look like:
```
 expect(venue.over_capacity?).to eq(false)
```